roles/rsyslog/tasks/outputs/files.yml

```diff
@@ -1,8 +1,50 @@
 ---
+- name: Create __rsyslog_conf_files_output_rules 0
+  set_fact:
+    __rsyslog_conf_files_output_rules:
+      [ { "name": "output-files-default", "type": "output",
+          "sections": [ { "options": "{{ lookup('template', 'output_files_default.j2') }}" } ] } ]
+  loop: '{{ rsyslog_output_files }}'
+  when: item.path is not defined
```

Comment:
So, if there are multiple items without a path, the default file would be repeated multiple times, is that intended?
Is it intended at all that the default is included whenever there is an item without a path?

Reply:
> So, if there are multiple items without a path, the default file would be repeated multiple times, is that intended?

Yes, it is repeated multiple times, but there will be just one output-files-default.conf.  I think there is no harm even if it is repeated.

> Is it intended at all that the default is included whenever there is an item without a path?

I think it's ok for the default conf file is generated since unless it is called from input roles with "call default_files", it does not affect the logging behaviour.